wireguard-monitor

```diff
@@ -8,6 +8,7 @@ ipColor="9dffa8"
 endpointColor="707eff"
 handshakeColor="a164e0"
 transferColor="f1e552"
+refreshCommand="xfce4-panel --plugin-event=${widgetName}:refresh:bool:true"
```

Comment:
${widgetName} not yet defined and result is wrong command and refreshCommand definition shall come after cmd parsing

```suggestion
refreshCommand=""
```

And in cmd parsing:

        -n|--name)
        refreshCommand="xfce4-panel --plugin-event=genmon-$2:refresh:bool:true"
        shift # past argument
        shift # past value